docker/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+echo Building container
+docker build -t gobuilder "$(pwd)"
+echo Running make script
+docker run --rm -v "/$(pwd)/.."://usr/src/myapp -w //usr/src/myapp gobuilder make
```

Comment:
This has been updated. And the GOPATH variable is /go by default (https://github.com/docker-library/golang/blob/2a15dfff04accfd31c2a45b3bb7423aa86aa2d60/1.8/jessie/Dockerfile line 45).